src/data/viteComboFetcher.ts

```diff
@@ -0,0 +1,129 @@
+/**
+ * **WARNING** This module is expected to be invoked from a Vite Plugin!
+ * 
+ * Aggregated all YAML Files in the ./combos Directory and interpretes 
+ */
+
+import { parse } from "yaml"
+import { DbCombosTableZod, DbTricksTableZod } from "../lib/database/schemas/CurrentVersionSchema"
+import { readFile } from "node:fs/promises"
+import { ZodError, z } from "zod"
+import {globby} from "globby"
+import { join } from "node:path"
+import { fileURLToPath } from "node:url"
+import { YamlComboTableSchemaZod } from "./viteSchemaGenerator"
+
+async function fetchYamlFile(path: string): Promise<z.infer<typeof DbCombosTableZod>> {
```

Comment:
Is there a reason why you specify the return type for this function but not for the others?

Reply:
hmm… good question. I think I set it during development to get the IDE to stop complaining :D I ill remove it